libs/sdk-ui/src/base/measureTitles/ignoreTitlesForSimpleMeasures.ts

```diff
@@ -0,0 +1,75 @@
+// (C) 2007-2020 GoodData Corporation
+import {
+    IInsightDefinition,
+    IBucket,
+    IAttributeOrMeasure,
+    isMeasure,
+    IMeasure,
+    IMeasureDefinitionType,
+    insightBuckets,
+    bucketItems,
+    isMeasureDefinition,
+    isArithmeticMeasureDefinition,
+    isPoPMeasureDefinition,
+    isPreviousPeriodMeasureDefinition,
+} from "@gooddata/sdk-model";
+
+function isAdhocMeasure(definition: IMeasureDefinitionType): boolean {
+    return (
+        isMeasureDefinition(definition) &&
+        (!!definition.measureDefinition.aggregation ||
+            !!definition.measureDefinition.computeRatio ||
+            (Array.isArray(definition.measureDefinition.filters) &&
+                definition.measureDefinition.filters.length > 0))
+    );
+}
+
+function isNonSimpleMeasure(definition: IMeasureDefinitionType): boolean {
```

Comment:
do we even need this? there is `isSimpleMeasure` type guard. so you can just negate that.

even if we need something like this, again it needs to go into sdk-model.

Reply:
Thanks, I've updated the PR.